public/app/features/alerting/unified/types/rule-form.ts

```diff
@@ -54,7 +54,7 @@ export interface RuleFormValues {
   contactPoints?: AlertManagerManualRouting;
   editorSettings?: SimplifiedEditor;
   metric?: string;
-
+  target_datasource_uid?: string;
```

Comment:
nit: in rule for values we use the naming convention without the _
example `is_paused` (in the dto type) vs  `isPaused`( in the rule form field)